sick-fits/frontend/components/styles/NavStyles.js

```diff
@@ -20,7 +20,7 @@ const NavStyles = styled.ul`
     cursor: pointer;
     @media (max-width: 700px) {
       font-size: 10px;
-      padding: 0 10px;
+      padding: 1rem;
```

Comment:
without some padding here, the hovered item underline appears under the dark bottom nav bar.